### ISSUE TITLE
Can only add 8 Tables in lobby

### ISSUE THREAD

**Issue by ideatrack** [PlayersCommittee/gemp-swccg-public#219]
I'm sure this is as designed, so may well be a feature request, but can you remove the max amount of tables that are able to be put up. I'm part of more than 4 leageus and would like to be able to put up a LS and Ds tabel for each to maximise playing time. Thanks

**Comment by EchoBaseTrooper** [comment]
Removing the table limit doesn't seem like a good idea. Increasing the limit to something higher than 8 might be okay.

**Comment by ideatrack** [comment]
Yeah, removing at all together is open to abuse, I agree. Upping it  a fair bit should work though. Thanks.

**Comment by ideatrack** [comment]
This would be really good if it could be set to say, 20

I expect that would be more than enough. There would defo. be more games played.

I hate having to choose!

Thanks!

**Comment by ideatrack** [comment]
Another call for this if possible, I think it would be great to open 2 tables (LS/DS) for every league you are in.

That would stop abuse, but means we don't have to pick some and not others.
